src/libraries/System.Private.Xml/src/System/Xml/Serialization/Compilation.cs

```diff
@@ -301,10 +301,14 @@ internal static bool GenerateSerializerToStream(XmlMapping[] xmlMappings, Type[]
                             continue;
                         }
 
+#if FEATURE_GAC
```

Comment:
Recommend instead using a single `#if` block instead of two blocks:

```cs
#if FOO
    if (condition)
#endif
    {
        /* logic */
    }
```